apps/frontend/src/components/global/RegistrationModal.vue

```diff
@@ -216,6 +216,7 @@ export default class RegistrationModal extends Vue {
             SnackbarModule.notify(
               'You have successfully registered a new user'
             );
+            ServerModule.FetchAllUsers();
```

Comment:
i'd appreciate you re-writing this function to use async/await notation as well instead of promise chaining